profile/src/main/java/com/scottlogic/deg/profile/reader/ConstraintReaderHelpers.java

```diff
@@ -44,18 +44,15 @@ public static Object getValidatedValue(ConstraintDTO dto) {
     public static Set<Object> getValidatedValues(ConstraintDTO dto) {
         if (dto.values == null) {
             throw new InvalidProfileException(String.format(
-                "Field [%s]: Couldn't recognise 'values' property, it must not contain 'null'",
-                dto.field
+                "Field [%s]: Requires a 'values' property but instead a 'value' property has been added.",
+                    dto.field
             ));
         }
 
         Set<Object> mappedValues = new HashSet<>();
         for (Object value : dto.values) {
             if (value == null) {
-                throw new InvalidProfileException(String.format(
-                    "Field [%s]: Set must not contain null",
-                    dto.field
-                ));
+                throw new InvalidProfileException(String.format("Field [%s]: Set must not contain null", dto.field));
```

Comment:
It seems like some auto-formatting has been applied here and elsewhere in this pull request. Unfortunately, this would make this file inconsistent with other files. Please could you revert this formatting?